packages/server/lib/open_project.js

```diff
@@ -73,6 +74,7 @@ const moduleFactory = () => {
           options.proxyServer = cfg.proxyUrl
           options.socketIoRoute = cfg.socketIoRoute
           options.chromeWebSecurity = cfg.chromeWebSecurity
+          options.downloadsFolder = path.join(cfg.projectRoot, 'cypress', 'downloads')
```

Comment:
I just noticed this got merged...

This should really be a configurable `cypress.json` option like the other folders are (such as videos/screenshots, etc)

Reply:
https://github.com/cypress-io/cypress/pull/14465